.github/workflows/build_package.yml

```diff
@@ -85,6 +101,7 @@ jobs:
 
     env:
       MANYLINUX_X86_64_IMAGE: ghcr.io/nod-ai/manylinux_x86_64:main
+      MANYLINUX_AARCH64_IMAGE: quay.io/pypa/manylinux_2_28_aarch64
```

Comment:
This is what is used [here](https://github.com/nod-ai/SRT/blob/b9dd7ee0404071415e8984bfe11406ec51ddb431/.github/workflows/build_package.yml#L85C9-L85C9)

Reply:
and also here: https://github.com/openxla/iree/blob/988f7c560f484640e910af6f284fa08565bf98dc/build_tools/python_deploy/build_linux_packages.sh#L67

Reply:
Added a comment about that. I can lift it out as a parameter you prefer (to have these dockers in one place)? LMKWYT